policies/taskrun_test.rego

```diff
@@ -0,0 +1,17 @@
+package hacbs.contract.taskrun
+
+test_allow_task_kind {
+	not allow_task_kind with data.kind as "taskblah"
+}
+
+test_allow_task_kind_denied {
+	allow_task_kind with data.kind as "TaskRun"
+}
+
+test_allow_task_name {
+	not allow_task_name with data.metadata as {"name": "mybadtask"}
+}
+
+test_sllow_task_name_denied {
```

Comment:
Typo "sllow"

Reply:
Also maybe line 11 should be "..._denied" this one should be "..._allowed" ..?

Reply:
Yes, I had those backwards. Fixed!